app/src/viewmodels/home/ProfileScreenViewModel.tsx

```diff
@@ -30,19 +33,20 @@ export function useProfileScreenViewModel(
 } {
   const userIsGuest = user.name === "Guest";
 
-  const [name, setName] = useState<string>(user.name);
+  const [name, sName] = useState<string>(user.name);
```

Comment:
Why the use of `sName` instead of `setName`?

Reply:
Okay I realised why. I understand the reasoning, however one the method should be renamed.